src/components/manifold-invoices/manifold-invoices.tsx

```diff
@@ -9,9 +11,7 @@ const $ = (amount: number, options: object = {}): string => {
     style: 'currency',
     currency: 'USD',
     ...options,
-  })
-    .format(amount / 100)
-    .replace(/\.00$/, '');
```

Comment:
I took out this formatting because while nice in one-off displays, it’s frustrating to read numbers formatted differently in a data table